longevity_test.py

```diff
@@ -35,6 +41,18 @@ class LongevityTest(ClusterTester, loader_utils.LoaderUtilsMixin):
     Test a Scylla cluster stability over a time period.
     """
 
+    def __init__(self, *args, **kwargs):  # pylint: disable=too-many-statements,too-many-locals,too-many-branches
+        super().__init__(*args)
+
+        # This ignores large_data warning messages "Writing large collection" for large collections to prevent
+        # creating SCT Events from these warnings.
+        # During large collections test thousands of warnings are being created.
+        self.validate_large_collections = self.params.get('validate_large_collections')
+        if self.validate_large_collections:
+            self.stack = contextlib.ExitStack()
+            self.stack.enter_context(ignore_large_collection_warning())
+            self.stack.enter_context(ignore_max_memory_for_unlimited_query_soft_limit())
```

Comment:
I wonder if we need this play with context - looking at the `BaseFilter` entering context means just `self.publish()` and in case we won't need cancel filtering we could just create filter and publish like this:
```
DbEventsFilter(
            db_event=DatabaseLogEvent.WARNING,
            line="Writing large collection"
        )).publish()
```
I think it should do the job and reduce LoC/complication

We could do it in test itself, so we enable it as a part of the test flow

Reply:
this style was mostly for purpose of reusing context manager we already have and used in other places.
also it can stopped if we want at later point.

I think it's align nicely the way we use those filters, and not having to keep both option in mind.

Reply:
if it's set like it is now, we can't stop at later point - playing with 'self.stack` to exit some context is quite complicated.

I'm just saying, this case context manager is more complicating than helping.